source/client-side-encryption/tests/README.rst

```diff
@@ -2693,302 +2694,343 @@ Create a ClientEncryption object named ``clientEncryption`` with these options:
 
 .. code:: typescript
 
-   ClientEncryptionOpts {
-      keyVaultClient: <keyVaultClient>;
-      keyVaultNamespace: "keyvault.datakeys";
-      kmsProviders: { "local": { "key": <base64 decoding of LOCAL_MASTERKEY> } }
+   class ClientEncryptionOpts {
+      keyVaultClient: <keyVaultClient>,
+      keyVaultNamespace: "keyvault.datakeys",
+      kmsProviders: { "local": { "key": <base64 decoding of LOCAL_MASTERKEY> } },
    }
 
 Create a MongoClient named ``encryptedClient`` with these ``AutoEncryptionOpts``:
 
 .. code:: typescript
 
-   AutoEncryptionOpts {
-      keyVaultNamespace: "keyvault.datakeys";
-      kmsProviders: { "local": { "key": <base64 decoding of LOCAL_MASTERKEY> } }
-      bypassQueryAnalysis: true
+   class AutoEncryptionOpts {
+      keyVaultNamespace: "keyvault.datakeys",
+      kmsProviders: { "local": { "key": <base64 decoding of LOCAL_MASTERKEY> } },
+      bypassQueryAnalysis: true,
    }
 
 The remaining tasks require setting ``RangeOpts``. `Test Setup: RangeOpts`_ lists the values to use for ``RangeOpts`` for each of the supported data types.
 
-Use ``clientEncryption`` to encrypt these values: 0, 6, 30, and 200. Ensure the type matches with the type of the encrypted field. For example, if the encrypted field is ``encryptedDoubleNoPrecision`` encrypt the value 6.0.
+Use ``clientEncryption`` to encrypt these values: 0, 6, 30, and 200. Ensure the type matches that of the encrypted field. For example, if the encrypted field is ``encryptedDoubleNoPrecision`` encrypt the value 6.0.
 
-Encrypt these values with the matching ``RangeOpts`` listed in `Test Setup: RangeOpts`_ and these ``EncryptOpts``:
+Encrypt using the following ``EncryptOpts``:
```

Comment:
The following text appears in an earlier paragraph in this section:

> The remaining tasks require setting ``RangeOpts``. Test Setup: RangeOpts lists the values to use for ``RangeOpts`` for each of the supported data types.

I don't think we needed to repeat the link. In all of the `EncryptOpts` examples that follow, I added `rangeOpts: <RangeOpts for Type>` to remind readers. I think that's more concise and ensure that `rangeOpts` appears in the `EncryptOpts` itself and is less likely to be missed.